### ISSUE TITLE
Can't get plugin to run

### ISSUE THREAD

**Issue by GeofoxCoding** [mjhea0/cypress-visual-regression#116]
Hello,

I just tested Cypress in a fresh Angular workspace successfully. Now I would like to be able to screenshot a part of my view and compare it to a given/existing screenshot.

This package here is the only one I found which seems to be compatible with my current cypress version 9.5.

I followed the setup instructions for typescript but I get an error on typescript startup:

 _An unexpected error occurred

Your pluginsFile threw an error from: C:\Users\Richard\Code\hwrm-ng\cypress\plugins\index.ts

TypeError: (0 , plugin_1.default) is not a function
    at configurePlugins (C:\Users\Richard\Code\hwrm-ng\cypress\plugins\index.ts:11:28)
    at C:\Users\Richard\AppData\Local\Cypress\Cache\9.5.0\Cypress\resources\app\packages\server\lib\plugins\child\run_plugins.js:94:12
    at tryCatcher (C:\Users\Richard\AppData\Local\Cypress\Cache\9.5.0\Cypress\resources\app\packages\server\node_modules\bluebird\js\release\util.js:16:23)
    at Function.Promise.attempt.Promise.try (C:\Users\Richard\AppData\Local\Cypress\Cache\9.5.0\Cypress\resources\app\packages\server\node_modules\bluebird\js\release\method.js:39:29)
    at load (C:\Users\Richard\AppData\Local\Cypress\Cache\9.5.0\Cypress\resources\app\packages\server\lib\plugins\child\run_plugins.js:91:7)
    at EventEmitter.<anonymous> (C:\Users\Richard\AppData\Local\Cypress\Cache\9.5.0\Cypress\resources\app\packages\server\lib\plugins\child\run_plugins.js:209:5)
    at EventEmitter.emit (node:events:520:28)
    at process.<anonymous> (C:\Users\Richard\AppData\Local\Cypress\Cache\9.5.0\Cypress\resources\app\packages\server\lib\plugins\util.js:19:22)
    at process.emit (node:events:520:28)
    at process.emit (C:\Users\Richard\AppData\Local\Cypress\Cache\9.5.0\Cypress\resources\app\node_modules\source-map-support\source-map-support.js:495:21)
    at emit (node:internal/child_process:938:14)
    at processTicksAndRejections (node:internal/process/task_queues:84:21)_

Can someone tell me if this package is right for what I'm trying to do and help me to solve the error?

**Comment by mjhea0** [comment]
Are you sure that error is coming from this library?

**Comment by GeofoxCoding** [comment]
Hi, yes I am.

As I wrote above I created a minimal new Angular workspace/app => works. I added cypress to the workspace and ran basic test => works.

My cypress/plugins/index.ts is nearly empty at this point. It only contains:
module.exports = (on, config) => {}

As soon as I follow your typescript integration instructions:
https://github.com/mjhea0/cypress-visual-regression#typescript

The error appears. If I switch back it works again.
